01-supp-factors.Rmd

```diff
@@ -134,24 +130,22 @@ summary(dat)
 
 Notice the `summary()` function handles factors differently to numbers (and strings), the occurence counts for each value is often more useful information.
 
-### Tip {.callout}
+> ## Tip {.callout}
 > 
 > The `summary()` function is a great way of spotting errors in your data, look at the *dat$Gender* column. It's also a great way for spotting missing data.
 
-### Challenge - Reordering factors {.challenge}
+> ## Challenge - Reordering factors {.challenge}
 > 
 > The function `table()` tabulates observations and can be used to create bar plots quickly. For instance:
-
-```{r using_factors}
+>
+~~~{.r}
```

Comment:
This is Markdown format for the chunk. Instead, please add `>` in front of the chunk. Recent versions of knitr will properly render the code chunk even though it is in  block quote, see this [issue](https://github.com/swcarpentry/lesson-template/issues/211).